### ISSUE TITLE
Cover ALL Plug Sockets in the Playroom

### ISSUE THREAD

**Issue by Stephanymtr** [bmhco/shared#20]
### **Task**
**Cover ALL Plug Sockets in the Playroom**
Plug socket covers prevent children from poking their fingers, or other objects into the plugs.  Some of the outlets aren’t covered because one is being used by the music keyboard, and others by the space heaters. 
- We are keeping the keyboard, so I’d suggest tying up the cable and covering the power plug. This will prevent children from unplugging/playing with the keyboard’s cable. 
- Will we be using the Wall-Mounted AC Unit to heat the room instead of the space/floor heaters when BMH opens?

> - If yes, I'd suggest removing the space heaters, so that we can cover all the plug sockets. 
> - If not, there are some follow up precautions that we'd need to take (e.g.  place it an area where the children can not reach it... or somewhere where it can not be knocked over or that will have a stand to prevent this from happening)

**Comment by nelsonic** [comment]
Indeed, at present the only sockets that don't have a cover are being used by heaters. 

The heaters are a _much_ more efficient way of heating the room than the AirCon.
But ... the moment we have other children in the room we should _either_:

a. Remove the free-standing air/fan heaters and use the AirCon to heat the room.
***or***
b. Install a wall-mounted heater and hide the socket it's connected to.

@iteles what is your preference on this? 

To cover the sockets in the Play Room we need to cover _all_ of the sockets on the first floor,
so corridors, kitchen, dining room, library. 
_Most_ of these are covered, but for the handful that aren't, we should check how other schools do it. 💭
